cassandra/src/main/scala/com/evolutiongaming/kafka/journal/cassandra/CreateKeyspace.scala

```diff
@@ -11,20 +11,20 @@ trait CreateKeyspace[F[_]] {
   def apply(config: KeyspaceConfig): F[Unit]
 }
 
-object CreateKeyspace { self =>
```

Comment:
IMO, its fine to keep `self` reference because Yaroslav used nested classes like `Main` in lots of places for simplifying navigation in IDE.

Its minor thing, feel free to ignore